### PR TITLE
cmake: Ensure that CMAKE_CODESIGN is set instead of trying to run empty commands.

### PR DESCRIPTION
**Pull request by erg**

### CODE REVIEW

**Comment by erg** [comment]
@swift-ci Please smoke test and merge

File: cmake/modules/AddSwift.cmake [hunk 1]
```diff
@@ -465,6 +465,10 @@ function(_add_swift_lipo_target)
   is_darwin_based_sdk("${LIPO_SDK}" IS_DARWIN)
   if(IS_DARWIN)
     if(LIPO_CODESIGN)
+      if(NOT CMAKE_CODESIGN)
+        message(FATAL_ERROR "CMAKE_CODESIGN is required, please set it")
```
Comment (hughbe) [inline cmake/modules/AddSwift.cmake]: Could use `precondition(CMAKE_CODESIGN MESSAGE "CMAKE_CODESIGN is required, please set it")

**Comment by erg** [comment]
@swift-ci Please smoke test

File: cmake/modules/AddSwift.cmake (hunk 1 above)
Comment (erg) [inline cmake/modules/AddSwift.cmake]: You're right, updating it..

**Comment by erg** [comment]
@swift-ci Please smoke test and merge

**Comment by slavapestov** [comment]
@swift-ci Please smoke test and merge

**Pull request merged**